jwcrypto/jwk.py

```diff
@@ -549,12 +550,84 @@ def import_from_pyca(self, key):
         else:
             raise InvalidJWKValue('Unknown key object %r' % key)
 
+    def import_from_pem(self, data, password=None):
+        """Imports a key from data loaded from a PEM file.
```

Comment:
So would you like a qualifier in the function signature?
Something like: import_from_pem(self, data, type=None|"private"|"public"|"x509", password=None) ?
Where None would be the default and autodetect what's in the file, but with the ability to force to check for a specific type ?